### PR TITLE
Updated Text 2 & 3

### PR DESCRIPTION
**Pull request by SandeepShenoyMicrosoft**
Please review the changes

### CODE REVIEW

File: text2.txt [hunk 1]
```diff
@@ -1 +1 @@
-Commit 2
\ No newline at end of file
+Again from Fork
```
Comment (SandeepShenoy72) [inline text2.txt:1]: How to verify this edit?

Comment (SandeepShenoyMicrosoft) [reply text2.txt:1]: It's validated

**Review by SandeepShenoy72 — APPROVED** [review]
Approved

**Comment by SandeepShenoy72** [comment]
Merging forked changes

**Pull request merged**